src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml

```diff
@@ -24,10 +24,26 @@
             <column name="enrollment_date" type="timestamp"/>
         </addColumn>
     </changeSet>
+
     <changeSet id="20211006170000-2" author="alexey@thehyve.nl">
         <addColumn tableName="subject_aud">
             <column name="enrollment_date" type="timestamp"/>
             <column name="enrollment_date_mod" type="BOOLEAN"/>
         </addColumn>
     </changeSet>
+
+    <changeSet id="add subject enrollment date" author="joris@thehyve.nl">
```

Comment:
I like this id. It makes me wonder why don't we identify other changesets is the same readable manner.

Reply:
I'm not sure either :-) I noticed this type of ID being used elsewhere and it's much more readable, also if looking through the liquidase change log table.